setup.py

```diff
@@ -28,30 +28,28 @@
     ],
     include_package_data=True,
     license='BSD',
-    install_requires=["Django>=2.2"],
+    install_requires=["Django>=3.2"],
```

Comment:
may want to consider `install_requires=["Django>=3.2,<=5.0"]`

Reply:
Same thought in https://github.com/django-guardian/django-guardian/pull/816#discussion_r1682714364

I would avoid upper bounds.

Reply:
Leaving it as is.